src/components/Header/Header.jsx

```diff
@@ -333,6 +353,11 @@ export function Header(props) {
                           {WEEKLY_SUMMARIES_REPORT}
                         </DropdownItem>
                       )}
+                      {canGetWeeklyVolunteerSummary && (
+                        <DropdownItem tag={Link} to="/totalorgsummary">
```

Comment:
In dark mode:

1. text color of menu item is black which is not correct as shown below:
![image](https://github.com/user-attachments/assets/29960e8c-48b9-43b3-af47-320496293560)

2. Hover over the menu item, highlight color is different and not consistent for new menu item:
**Color of highlight for new menu item is:** 
<img width="1509" alt="image" src="https://github.com/user-attachments/assets/f9484b5e-d7c7-437b-a3af-3b33ab156532">

**Color of highlight for existing menu item is:** 
<img width="1483" alt="image" src="https://github.com/user-attachments/assets/4e2faa96-ee55-40eb-8a29-3d5488b567ed">

Reply:
Hello @cgomezhub , This observation is still occurring after latest changes

Reply:
Resolving the conversation, working fine after latest changes in [1a2132e](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2503/commits/1a2132e0146cf41168d15756c0587fb267d42ee5)

Reply:
it's done. Thanks!